clinical/src/main/kotlin/com/hartwig/actin/clinical/feed/standard/DataQualityMask.kt

```diff
@@ -10,6 +11,21 @@ private fun ProvidedPatientRecord.scrubModifications() =
 private fun ProvidedPatientRecord.scrubMedications() =
     this.copy(medications = null)
 
+private fun ProvidedPatientRecord.useOnlyPriorOtherConditions(clinicalConfiguration: ClinicalConfiguration) =
+    if (clinicalConfiguration.useOnlyPriorOtherConditions) {
+        this.copy(
+            treatmentHistory = emptyList(),
+            complications = emptyList(),
+            surgeries = emptyList(),
+            toxicities = emptyList(),
+            priorPrimaries = emptyList(),
+            allergies = emptyList(),
+            tumorDetails = this.tumorDetails.copy(diagnosisDate = null, lesionSite = null, lesions = emptyList())
+        )
+    } else {
+        this
+    }
```

Comment:
With the config being checked below, this can be simplified to:
```suggestion
private fun ProvidedPatientRecord.useOnlyPriorOtherConditions() = this.copy(
    treatmentHistory = emptyList(),
    complications = emptyList(),
    surgeries = emptyList(),
    toxicities = emptyList(),
    priorPrimaries = emptyList(),
    allergies = emptyList(),
    tumorDetails = this.tumorDetails.copy(diagnosisDate = null, lesionSite = null, lesions = emptyList())
)
```